doc/source/reference/maskedarray.baseclass.rst

```diff
@@ -417,8 +417,6 @@ Container customization: (see :ref:`Indexing <arrays.indexing>`)
    MaskedArray.__getitem__
    MaskedArray.__setitem__
    MaskedArray.__delitem__
-   MaskedArray.__getslice__
-   MaskedArray.__setslice__
```

Comment:
Right, but without the `__`. I was under the impression that was just a coincidental reference to something else. If that's not the case, I'll wipe that one too